www/templates/www/custominvoice.html

```diff
@@ -36,7 +40,7 @@ <h2>{% trans 'Pending invoices'%}</h2>
     <td>{{ custominvoice.days }} {% trans 'days of' %} {{ custominvoice.subscription.service.name }}</td>
     <td>{{ custominvoice.amount }}</td>
     <td>{{ custominvoice.reference_number }}</td>
-    <td><a href="{% url 'custominvoice_action' action='delete' invoiceid=custominvoice.id %}">{% trans 'delete' %}</a></td>
+    <td><a onclick="return confirm('{% blocktrans %}Are you sure? Please note, if you have already paid the invoice but the payment is not registered yet deleting the custom invoice will never be targeted{% endblocktrans %}')" href="{% url 'custominvoice_action' action='delete' invoiceid=custominvoice.id %}">{% trans 'delete' %}</a></td>
```

Comment:
I think this sentence could use a comma ("...but the payment is not registered yet, deleting the..."). I also don't understand what "targeted" means here.

Reply:
"kohdistua", couldn't think of a better word...

We've had at least few persons who created custom invoice on day 1, paid on day2 and deleted on day3 and then bank transactions were read in on day 7 or something like that but of course that wont mark anything for the user and is just considered a donation at that point...

Reply:
Oh, I see. Something like "Please note: deleting a custom invoice before the payment for it is processed by the system will cause the payment to fail", then?

Reply:
well the payment will still come through but it wont map to the user and will just be a donation :D 
anyways I removed the whole inline js part as Olmari reminded me why we don't do that :D